CONTRIBUTING

```diff
@@ -19,8 +19,16 @@ guidelines:
     1. Anything other than a trivial contribution requires a Contributor
     License Agreement (CLA), giving us permission to use your code. See
     https://www.openssl.org/policies/cla.html for details.  If your
-    contribution is too small to require a CLA, put "CLA: trivial" on a
-    line by itself in your commit message body.
+    contribution is too small to require a CLA (e.g. fixing a spelling
+    mistake), place the text "CLA: trivial" on a line by itself separated by
```

Comment:
> The sentence: "If your contribution is too small ... body of the commit message" is quite a mouthful.

Maybe a comma between "itself" and "separated by" would help breaking the sentence up?

Also, you could shorten "the rest of the body of the commit message" to "the rest of the commit message", as it was @levitte's [original suggestion](https://github.com/openssl/openssl/pull/9265/files/679cb0a6f4985f63ee2fc9aae3a666a43fd219b9#r298472820).

Reply:
Going with @levitte's suggestion.  There are enough commas in the sentence already :)